Analysis/Utkscan/core/source/TimingCalibrator.cpp

```diff
@@ -32,7 +32,7 @@ TimingCalibrator* TimingCalibrator::get() {
 }
 
 void TimingCalibrator::ReadTimingCalXml() {
```

Comment:
This should follow the same programing as the Notebook since it is reopening the same .xml file again.